integration_test.go

```diff
@@ -0,0 +1,387 @@
+// Package viamcartographer_test tests the functions that require injected components (such as robot and camera)
```

Comment:
[minor] The comment here should be primarily focus on the type of tests this package runs, i.e. integration tests that test the full pipeline from providing fake data from an injected camera to a viam-cartographer binary and hitting the endpoints to confirm proper operation. Addition info such as using the internal package strikes me as superfluous.